zwave_js_server/const.py

```diff
@@ -653,6 +653,8 @@ class HeatingScale(IntEnum):
 
 CoolingScale = HeatingScale
 
+MeterScaleType = Union[CoolingScale, ElectricScale, GasScale, HeatingScale, WaterScale]
+
 METER_TYPE_TO_SCALE_ENUM_MAP = {
```

Comment:
Maybe we need to type this dict?

Reply:
this is what I get when I try:
```
zwave_js_server/const.py:659: error: Dict entry 0 has incompatible type "MeterType": "Type[ElectricScale]"; expected "MeterType": "Union[HeatingScale, ElectricScale, GasScale, WaterScale]"
zwave_js_server/const.py:660: error: Dict entry 1 has incompatible type "MeterType": "Type[GasScale]"; expected "MeterType": "Union[HeatingScale, ElectricScale, GasScale, WaterScale]"
zwave_js_server/const.py:661: error: Dict entry 2 has incompatible type "MeterType": "Type[WaterScale]"; expected "MeterType": "Union[HeatingScale, ElectricScale, GasScale, WaterScale]"
zwave_js_server/const.py:662: error: Dict entry 3 has incompatible type "MeterType": "Type[HeatingScale]"; expected "MeterType": "Union[HeatingScale, ElectricScale, GasScale, WaterScale]"
zwave_js_server/const.py:663: error: Dict entry 4 has incompatible type "MeterType": "Type[HeatingScale]"; expected "MeterType": "Union[HeatingScale, ElectricScale, GasScale, WaterScale]"
```

any suggestions on how to fix that?

Reply:
Maybe we need to use a TypedDict?

Reply:
It won't recognize the enum value as a key

Reply:
I needed to add `Type` as a wrapper 🤦🏾